rmgpy/solver/surfaceTest.py

```diff
@@ -276,3 +276,233 @@ def test_solve_ch3(self):
 
         # Check that we've reached equilibrium by the end
         self.assertAlmostEqual(reaction_rates[-1, 0], 0.0, delta=1e-2)
+
+    def test_solve_h2_coverage_dependence(self):
+        """
+        Test the surface batch reactor can properly apply coverage dependent parameters
+        with the dissociative adsorption of H2.
+
+        Here we choose a kinetic model consisting of the dissociative adsorption reaction
+        H2 + 2X <=> 2 HX
+        We use a SurfaceArrhenius for the rate expression.
+        """
+        h2 = Species(
+            molecule=[Molecule().from_smiles("[H][H]")],
+            thermo=ThermoData(Tdata=([300, 400, 500, 600, 800, 1000, 1500], "K"),
+                              Cpdata=([6.955, 6.955, 6.956, 6.961, 7.003, 7.103, 7.502], "cal/(mol*K)"),
+                              H298=(0, "kcal/mol"),
+                              S298=(31.129, "cal/(mol*K)")))
+        x = Species(
+            molecule=[Molecule().from_adjacency_list("1 X u0 p0")],
+            thermo=ThermoData(Tdata=([300, 400, 500, 600, 800, 1000, 1500], "K"),
+                              Cpdata=([0., 0., 0., 0., 0., 0., 0.], "cal/(mol*K)"),
+                              H298=(0.0, "kcal/mol"),
+                              S298=(0.0, "cal/(mol*K)")))
+        hx = Species(
+            molecule=[Molecule().from_adjacency_list("1 H u0 p0 {2,S} \n 2 X u0 p0 {1,S}")],
+            thermo=ThermoData(Tdata=([300, 400, 500, 600, 800, 1000, 1500], "K"),
+                              Cpdata=([1.50, 2.58, 3.40, 4.00, 4.73, 5.13, 5.57], "cal/(mol*K)"),
+                              H298=(-11.26, "kcal/mol"),
+                              S298=(0.44, "cal/(mol*K)")))
+
+        rxn1 = Reaction(reactants=[h2, x, x],
+                        products=[hx, hx],
+                        kinetics=SurfaceArrhenius(A=(9.05e18, 'cm^5/(mol^2*s)'),
+                                                  n=0.5,
+                                                  Ea=(5.0, 'kJ/mol'),
+                                                  T0=(1.0, 'K'),
+                                                  cov={'*': {'E': (0.0, 'J/mol'), 'm': -1.0, 'a': 0.0}}))
+
```

Comment:
should this be `*` or `[Pt]` for empty site?  you can make a an empty site RMG molecule using `*` as SMILES, but when you convert the molecule to smiles you get `[Pt]`, so I think it's `[Pt]` we want here

Reply:
I don't think it should be specified `[Pt]` especially if this reaction wasn't from a Pt surface. There will be a lot in the future (hopefully) that refer to empty sites that are not Pt